lib/cretonne/src/verifier.rs

```diff
@@ -261,12 +232,7 @@ impl<'a> Verifier<'a> {
                 self.verify_sig_ref(inst, sig_ref)?;
                 self.verify_value_list(inst, args)?;
             }
-            // These have no entity references to check. Explicitly listed so we get a compile error
-            // if more variants are ever added.
-            &Nullary { .. } |
-            &UnaryImm { .. } |
-            &UnaryIeee32 { .. } |
-            &UnaryIeee64 { .. } => {}
+            _ => {}
```

Comment:
I think it is a good idea to keep the exhaustive list of instruction formats here and no do a catch-all match. It is pretty common to add new formats, and I keep forgetting what needs to be updated for a new format.